lib/AST/ASTVerifier.cpp

```diff
@@ -2211,6 +2211,22 @@ class Verifier : public ASTWalker {
         }
       }
 
+      if (var->getAttrs().hasAttribute<ImplicitlyUnwrappedOptionalAttr>()) {
+        auto ty = var->getInterfaceType();
+
+        // Look through storage modifier.
+        if (auto weakTy = ty->getAs<ReferenceStorageType>())
+          ty = weakTy->getReferentType();
```

Comment:
You can replace these two lines with getReferenceStorageReferent(), which is a no-op if its not a reference storage type

Reply:
Great, thanks!